Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1502,7 +1502,7 @@ private void AddCorners()
                 ApplyMaterialToAllRenderers(cornerVisual, handleMaterial);
 
                 AddComponentsToAffordance(corner, new Bounds(cornerbounds.center * invScale, cornerbounds.size * invScale), RotationHandlePrefabCollider.Box, CursorContextInfo.CursorAction.Scale, scaleHandleColliderPadding);
-                corners.Add(corner.transform);
+                corners.Add(cornerVisual.transform);
```

Comment:
there's two things about this:
1. we're trying to get away from bounding box and use bounds control instead - fixes made in boundingbox should also be made in boundscontrol (in non critical cases I would even recommend doing the changes only in bounds control to avoid having to fix them in two spots)
2. I'm very certain that this will break a lot of the boundscontrol behavior on reloading visuals on config changes - i don't have all these changes published yet but this definitely will have consequences on all the rig reloading :-/

Reply:
I'll look into ways to avoid changing the behavior of the corners variable then

Reply:
resolving this as we had a offline chat about this - boundscontrol behavior will be checked separately